src/applications/disability-benefits/all-claims/pages/secondaryIncidentAuthorities.js

```diff
@@ -40,6 +40,10 @@ export const uiSchema = index => ({
           zipCode: {
             'ui:title': 'Postal Code',
             'ui:validations': [validateZIP],
+            'ui:errorMessages': {
+              pattern:
+                'Please enter a valid 5- or 9-digit Postal code (dashes allowed)',
```

Comment:
I think we normally omit the dashes in this message ("5 or 9 digit").

Reply:
@jbalboni this is used on ~4 other pages, so I think we originally pulled from one of those examples:
![image](https://user-images.githubusercontent.com/22040793/51135913-2372a900-1809-11e9-9e29-20c4c0ac236a.png)

Reply:
I think you can leave it as is for now and create a ticket for @peggygannon to check and see if it needs to be changed across the form going forward. I believe other forms omit the dashes, but I'm not sure it was intentionally done a different way on this form.